app/safe/internal/bootstrap/k8s.go

```diff
@@ -21,6 +23,16 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+var podNamespace string
+var id string = "VSECMSAFE"
+func init() {
```

Comment:
I have mixed feelings about this `init()` function.; but we can keep it for now.

If the pattern repeats, we might want to centralize it.

I might create a follow-up PR maybe later on.  But it fits the bill for now.

Also +1 for crashing if namespace not found. it’s better not to assume a default here.

Reply:
💯 Agree on keeping it centeralized, I kept it here since we don't need namespace any other place currently.
Having said, even I'm inclined to keep it centeralized which can be consumed anywhere I'll make that change and re-submit for review. 👍